.github/workflows/code-scanning.yml

```diff
@@ -0,0 +1,36 @@
+name: Scan code for vulnerabilities
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    # run 00:00 every Monday
+    - cron: '00 00 * * 1'
```

Comment:
Can we change this to 05:00 on Monday - that way there will be less time between a reported issue and us actioning it.

Reply:
👍